src/services/product.services.new.test.ts

```diff
@@ -0,0 +1,74 @@
+import { faker } from '@faker-js/faker';
+import { db } from '../../prisma/db';
+import { ApiError } from '../config/apiError';
+import { errors } from '../config/errors';
+import { ProductSerializer } from '../serializers/product-serializer';
+import { productRandomRaw } from '../test/test-constants';
+import { UpdateProductRequest } from '../types/product';
+import { ProductService } from './product.services';
+
+describe('Products Service', () => {
+  describe('Update Product Service', () => {
```

Comment:
What is the reason for creating these two descriptive sections? I think we can consolidate them into one with the text 'Update Product Service' since the file is already intended for product service

Reply:
True. No reason really.I Will the external describe